migration/sql-files/002-fix-273-workitemtype.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE work_item_types
+SET fields = '{"system.state": {"Type": {"Kind": "enum", "Values": ["new", "in progress", "resolved", "closed"], "BaseType": {"Kind": "string"}}, "Required": true}, "system.title": {"Type": {"Kind": "string"}, "Required": true}, "system.creator": {"Type": {"Kind": "user"}, "Required": true}, "system.assignee": {"Type": {"Kind": "user"}, "Required": false}, "system.description": {"Type": {"Kind": "string"}, "Required": false}}';
```

Comment:
Hi @aslakknutsen 
Made code changes to reflect the above design where I've added support for updating common types here :
https://github.com/almighty/almighty-core/pull/264/files#diff-fe79cd9ae6e3b75f55aa2640b65d3d77R219

Could you please review and let me know if there are any more changes needed for this PR.